site/api/components/Class/Constructors.tsx

```diff
@@ -13,13 +16,15 @@ export function Constructors({
   if (!ctors.length) {
     return null;
   }
-  const items = ctors.map(({ accessibility, name, params }, i) => (
+  return ctors.map((v) => (
     <>
-      {accessibility ? `${accessibility} ` : undefined}
-      <span style="color: rgb(98, 232, 132);">{name}</span>(
-      <Params getLink={getLink}>{params}</Params>);
-      {"\n"}
+      <CodeBlock>
+        {v.accessibility ? <StyleKw>`${v.accessibility} `</StyleKw> : undefined}
```

Comment:
```suggestion
        {v.accessibility ? <StyleKw>{v.accessibility}{" "}</StyleKw> : undefined}
```

Reply:
That would still style the space with the wrong colour … is there anything else that this improves?

Reply:
@KnorpelSenf No one sees the color of the spaces. You should fix it all other places if you are able to see the color of the spaces.